### PR TITLE
Closes #3 - Create initial index.html markup

### LINKED ISSUE #3: Create initial index.html markup

### PR DESCRIPTION
**Pull request by VitaliiMaksymiv**

### CODE REVIEW

**Review by Vladacdc — APPROVED** [review]

**Review by SviatoslavBachynskyi — APPROVED** [review]

**Review by konerokua — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
```
Comment (VitalikRibiy) [inline index.html]: Close body plez

**Review by konerokua — APPROVED** [review]
Molodec

File: index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body> 
+</body>
+</html>
```
Comment (BohdanBuhrii) [inline index.html]: delete enter in last row

**Review by VasilDun — APPROVED** [review]

**Review by BohdanBuhrii — CHANGES_REQUESTED** [review]

**Review by andriyshevchenko — COMMENTED** [review]
a

**Review by VitalikRibiy — CHANGES_REQUESTED** [review]

**Review by andriyshevchenko — APPROVED** [review]
nice

**Review by BohdanBuhrii — APPROVED** [review]

**Review by Vladacdc — APPROVED** [review]
Нормас

**Review by VitalikRibiy — APPROVED** [review]

**Review by andriyshevchenko — APPROVED** [review]
-_-

**Review by VasilDun — APPROVED** [review]

**Review by konerokua — APPROVED** [review]
molodec, dvagdy

**Pull request merged**